src/store/modules/sidenavBoxes.js

```diff
@@ -0,0 +1,39 @@
+function initialState () {
+  return {
+    mapCollapsed: false,
+    storeCollapsed: false,
+    storesCollapsed: false,
+    groupCollapsed: false,
+  }
+}
+
+export default {
+  namespaced: true,
+  state: initialState(),
+  getters: {
+    mapCollapsed: state => state.mapCollapsed,
+    storeCollapsed: state => state.storeCollapsed,
+    storesCollapsed: state => state.storesCollapsed,
+    groupCollapsed: state => state.groupCollapsed,
+  },
+  actions: {
```

Comment:
I'm unsure if we should keep the mix of mutations and actions as interface. It seems unnecessary boilerplate to keep the actions as extra layer (you can also `commit('sidenavBoxes/hideGroup')`), on the other hand we mostly used actions in other modules.

Actions have the advantage that they can easily be extended and also call other actions and mutations. The downside is a bit more boilerplate. Should we make a policy here to only have actions as interface? (This also affects the `map` module). @nicksellen your opinion would be valuable here!

Reply:
I would be inclined to keep to just exposing getters + actions in the store modules. Boilerplate can be reduced by writing functions that generate it. In this case the getters, actions, and mutations could all be generated.

Reply:
Something like this could be nice:

```
import { toggles, switches } from '@/stores/helpers'
function initialState () {
  return {
    map: true,
    store: false,
    stores: true,
    group: true,
  }
}

export default {
  namespaced: true,
  state: initialState(),
   // actions: toggleMap; getters: showMap etc
  ...toggles(['map', 'store', 'stores']), 

   // actions: hideGroup, showGroup; getters: showGroup
  ...switches(['group']),
}
```

- no need to duplicate the 'sidenav' name inside the 'sidenav' module
- getter names should be positive: 'showMap', 'showGroup' (reduces cognitive burden of double negation: `mapCollapsed: false`)
- defaults get set via initialState

What do you think? Could be fun to code a helper like this :)

Reply:
I created #819 - let's go ahead and merge, later refactor.